webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -460,6 +478,10 @@ Kommunicate.richMsgEventHandler = {
     },
     handleFormSubmit: function(e) {
         e.preventDefault();
+    },
+    isValidString: function (str, value) {
+        var regex = new RegExp(str);
+        return regex.test(value);
```

Comment:
do we need a variable here, or we can directly write new RegExp(str).test(value)

Reply:
@devashishmamgain  regex variable is removed